manual_jb/content/analyses/wfn_functionality.md

```diff
@@ -0,0 +1,78 @@
+# Water facility network functionality
+
+**Description**
+
+This analysis computes the functionality of water facility networks.
+
+The computation uses inputs from Monte Carlo failure analysis using water facility damage information, as well as 
+pipeline functionality data to determine the functionality probability and failure states for a corresponding water 
+facility network by performing a reachability analysis.
+
+The output of the analysis consists of a CSV with functionality probabilities for water facilities, and a CSV 
+with functionality failure states.
+
+**Contributors**
+
+- Science: Neetesh Sharma, Armin Tabandeh, Paolo Gardoni
+- Implementation: Neetesh Sharma, Armin Tabandeh, Santiago Núñez-Corrales, and NCSA IN-CORE Dev Team
+
+**Related publications**
+
+* Sharma, N., Tabandeh, A., & Gardoni, P. (2019). Regional resilience analysis: A multi-scale approach to model the 
+  recovery of interdependent infrastructure. In P. Gardoni (Ed.), *Handbook of sustainable and resilient 
+  infrastructure* (pp. 521–544). New York, NY: Routledge.
+* Sharma, N., Tabandeh, A., & Gardoni, P. (2020). Regional resilience analysis: A multi-scale approach to optimize 
+  the resilience of interdependent infrastructure. *Computer‐Aided Civil and Infrastructure Engineering*, **35(12)**, 
+  1315-1330.
+* harma, N., & Gardoni, P. (2022). Mathematical modeling of interdependent infrastructure: An object-oriented 
```

Comment:
Typo? Missing a S? Sharma?

Reply:
@longshuicy can you take another look, I think this issue is addressed.